### DISCUSSION TITLE
SSL not working

### DISCUSSION THREAD

**Discussion by Jeffer1980** [vercel/community#2555] [Help] [answered]
### Summary

I have my domain on GoDaddy. I deployed my project on Vercel and the auto-generated link works fine, but the domain I connected to the project gives SSL issues. 
Vercel is telling me DNS is configured correctly, but (tried with different browsers) no https connection is working. I configured TXT, A, CNAME and CAA records.
When I check the certificate of the custom domain, the certificate is not issued by LetsEncrypt but by Starfield technologies.

Anybody any idea how I can have Vercel generate the right certificates?

**Comment by amyegan** [comment, marked as answer]
Hi @Jeffer1980. If you share the domain or project id, I can use that to look into what's going wrong.

**Reply by Jeffer1980** [reply]
Now it works... looks like the DNS changes were not yet propagated
